bundles/org.openhab.binding.bluetooth.hdpowerview/README.md

```diff
@@ -0,0 +1,81 @@
+
+The discovery process can be started by pressing the `+` button at the lower right of the Main UI Things page, selecting the Bluetooth binding, and pressing `Scan`.
+Any discovered shades will be displayed with the name prefix 'Powerview Shade'.
+
+## Configuration
+
+| Configuration Parameter | Type               | Description                                                                                                         |
+|-------------------------|--------------------|---------------------------------------------------------------------------------------------------------------------|
+| address                 | Required           | The Bluetooth MAC address of the shade.                                                                             |
+| bleTimeout              | Optional, Advanced | The maximum number of seconds to wait before transactions over Bluetooth will time out (default = 6 seconds).       |
+| heartbeatDelay          | Optional, Advanced | The scanning interval in seconds at which the binding checks if the Shade is on- or off- line (default 15 seconds). |
+| pollingDelay            | Optional, Advanced | The scanning interval in seconds at which the binding checks the battery status (default 300 seconds).              |
+| encryptionKey           | Optional           | The key to be used when encrypting commands to the shade. See [next chapter](#encryption-key).                      |
+
+## Encryption Key
+
+If you want to send position commands to a shade, then an encryption key may be required.
+If the shade is NOT included in the Powerview App, then an encryption key is not required.
+But if it IS in the Powerview App, then openHAB has to use the same encryption key as used by the App.
+Currently you can only discover the encryption key by snooping the network traffic between the App and the shade.
+Please post on the openHAB community forum for advice about how to do this.
+
+## Channels
+
+A shade always implements a roller shutter channel `position` which controls the vertical position of the shade's (primary) rail.
+If the shade has slats or rotatable vanes, there is also a dimmer channel `tilt` which controls the slat / vane position.
+If it is a dual action (top-down plus bottom-up) shade, there is also a roller shutter channel `secondary` which controls the vertical position of the secondary rail.
+
+| Channel       | Item Type      | Description                                           |
+|---------------|----------------|-------------------------------------------------------|
+| position      | Rollershutter  | The vertical position of the shade's rail.            |
+| secondary     | Rollershutter  | The vertical position of the secondary rail (if any). |
+| tilt          | Dimmer         | The degree of opening of the slats or vanes (if any). |
+| battery-level | Number:Percent | Battery level (10% = low, 50% = medium, 100% = high). |
```

Comment:
`Number:Percent` is an unsupported type, i guess you mean `Number:Dimensionless`

Reply:
Yup.